src/main/java/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package service.impl;
+
+import db.Storage;
+import java.util.Map;
+import service.ReportService;
+
+public class ReportServiceImpl implements ReportService {
+    public static final String BANANA = "banana";
+    public static final String APPLE = "apple";
+
+    @Override
+    public String createReport() {
```

Comment:
Do logic here more independently. The key in the map is equal to the fruit name. You can use `entry.getKey()`, instead of creat additional constant and if-statements